mne_bids/write.py

```diff
@@ -978,6 +980,15 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
         also be a string pointing to a .trans file containing the
         transformation matrix. If None, no sidecar JSON file will be written
         for `t1w`
+    deface : bool | dict
+        If False, no defacing is performed.
+        If True, deface with default parameters.
+        If dict, accepts the following keys:
+            `inset`: how far back as a fraction of mri space to start
+                     defacing relative to the nasion (default 0.2)
```

Comment:
does this 0.2 value match to centimeters or some meaningful unit?

Reply:
can you just clarify what 0.2 means. What is fraction of mri space?

Reply:
0.2 is a fraction of field of view of the mri, I'll look into changing it, I think it can be expressed in metric units

Reply:
It doesn't look like RAS has sensible units... the head is 60 something which is about double what it should be in cm. Is it better as a fraction or as a somewhat arbitrary unit? I'll keep trying to figure out what the units are actually

Reply:
It's in mm, changed accordingly!